hw/ip/alert_handler/doc/_index.md

```diff
@@ -935,16 +935,6 @@ In order to generate the register file for four alert sources, from
 $ ./reg_alert_handler.py alert_handler.hjson.tpl -n 4 > alert_handler.hjson
 ```
 
-Note that you should also update the regfile wrapper after updating the
-regfile using
-
-```console
-$ ./alert_handler_reg_wrap.sv.tpl -n 4 > ../rtl/alert_handler_reg_wrap.sv
-```
-
-{{< registers "hw/top_earlgrey/ip/alert_handler/data/autogen/alert_handler.hjson" >}}
```

Comment:
This line is producing the register table and needs to stay.

Reply:
Handling PRs during a meeting is not always a good idea :see_no_evil: